ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -466,7 +476,9 @@ def query_with_retry(ctx: Context, query: str, timeout: int, retries: int) -> bo
     return True
 
 
-def reattach_partition(ctx: Context, table: str, partition: str) -> bool:
+def reattach_partition(
```

Comment:
Function `reattach partition` with flag `detach`, a bit confusing :(
```suggestion
def handle_partition(ctx: Context, table: str, partition: str, detach: bool, attach: Bool)
```